### ISSUE TITLE
Q: any plan for a version compatible with ubuntu xenial 16.04?

### ISSUE THREAD

**Issue by pine3ree** [Ximi1970/systray-x#19]
thanks!

**Comment by Ximi1970** [comment]
You're in luck 16.04 is still available at OBS. Enabled the build but it failed. Needs some small tlc I think. It is on the list!

**Comment by Ximi1970** [comment]
Oke fixed in the feature-ubuntu branch. Cannot test it until I install Ubuntu 16.04. System is now quiet busy installing Gentoo and Arch.

**Comment by pine3ree** [comment]
Hello @Ximi1970,
I build successfully (ubuntu-feature branch) the xpi extension and the companion app. I had to copy the app into /usr/local/bin (missing instruction in README) and change its path in the installed json file to match the position where I put it. Now The icon appears but I get a version mismatch warning.
SHow/Hide works, but I do not see the unread messages count. kind regards.

**Comment by pine3ree** [comment]
pinging @Ximi1970 ...I noticed that just before the warning popover I see the message count that disappears in less than 1 second....

**Comment by pine3ree** [comment]
Unread messages update: it seems the matter for the unread message count is different.
I have many pop accounts all using the Local Folders as the message storage. I also have many (I mean a lot) filters that dispatch incoming messages to those subfolders. And I have 1 separate imap account.
The unread count works only for the imap account, although in the systray-x prefs I can see all pop-account and the "local folders" folder checked.

I also noticed another difference from the old FireTray: systray-x seems to work only based on a customizable poll interval, while fire-tray reacted instantly to an change in a message status. If I marked any message as unread, that would be reflected on the tray-icon. Is this supported by the new TB extension api?  

Thanks and kind regards.
